tests/Unit/AccountTest.php

```diff
@@ -31,20 +32,23 @@ public function test_it_has_many_genders()
 
     public function test_it_has_many_notifications()
     {
-        $account = factory('App\Account')->create([]);
+        $contact = factory(Contact::class)->create();
```

Comment:
I don't understand why you've changed this kind of things. What was wrong with `App\Account`? At least you don't need to import all the classes at the top of the test files that way.

Reply:
Because it makes it easier to rename the class, if that ever happens. The issue with using the string approach is that you MUST know that they are used, while an IDE such as PHPStorm will make it easy to rename a class and all its references.

Reply:
Interesting, thanks for the explanation